ui/admin/tests/acceptance/host-catalogs/delete-test.js

```diff
@@ -84,9 +89,7 @@ module('Acceptance | host-catalogs | delete', function (hooks) {
 
     await click(`[href="${urls.hostCatalog}"]`);
 
```

Comment:
u also need to to add `await click(MANAGE_DROPDOWN_SELECTOR)` before asserting that the delete action does not exist.

Reply:
Completely missed that! Thank u lis🤗